src/context-providers/builder/element.types.ts

```diff
@@ -34,6 +34,12 @@ export interface IConditionalRendering {
     internalId: string;
 }
 
+export interface IPrefill {
+    prefillMode?: string;
```

Comment:
Should this be some string literal? What are the values?

Reply:
act the value for this is either 'Myinfo' or 'Previous source'

Reply:
let's go for string literal, in the long run, we will go with discriminating union

```suggestion
    prefillMode?:"Myinfo"|"Previous source";
```